webhook/client/client.go

```diff
@@ -16,14 +17,15 @@ import (
 	"github.com/longhorn/longhorn-manager/datastore"
 	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
 	lhinformers "github.com/longhorn/longhorn-manager/k8s/pkg/client/informers/externalversions"
+	"github.com/longhorn/longhorn-manager/types"
 )
 
 type Client struct {
 	clients.Clients
 	Datastore *datastore.DataStore
 }
 
-func New(ctx context.Context, config *rest.Config, namespace string) (*Client, error) {
+func New(ctx context.Context, config *rest.Config, namespace, serverType string) (*Client, error) {
```

Comment:
NIT:
I just realize that this package name is a little bit confusing. It's more than the Longhorn client used in the webhook service rather than a webhook client. But we will use it to start the webhook service in `xxxxxxxWebhookListenAndServe` as well...